fe/fe-core/src/main/java/com/starrocks/catalog/Replica.java

```diff
@@ -90,6 +90,8 @@ public enum ReplicaStatus {
     // the version could be queried
     @SerializedName(value = "version")
     private volatile long version;
+    @SerializedName(value = "lastModificationTime")
+    private long lastModificationTime;
```

Comment:
The name `lastModificationTime` is a bit too broad, requiring more context to understand its meaning. How about `lastStatisticsUpdateTime`?